book/src/02_development/03_local_release.md

```diff
@@ -0,0 +1,10 @@
+# Local release
+
+These are instructions for releasing the [EDR NPM package](../../crates/edr_evm_napi/package.json) locally for debugging purposes.
+
+1. Install and start [Verdaccio](./02_verdaccio.md).
+2. Go to the [edr_evm_napi](../../crates/edr_evm_napi/) directory.
+3. Run `yarn build`.
+4. Look for the NAPI binary that was built for your platform. It has the format `edr.<PLATFORM>.node`. For example on Apple Silicon Macs, it's called `edr.darwin-arm64.node`.
+5. Move the NAPI binary to the appropriate platform-specifc package in the [npm](../../crates/edr_evm_napi/npm) directory. For example, on Apple Silicon Macs: `mv edr.darwin-arm64.node npm/darwin-arm64`.
+6. Complete the Verdaccio [publish steps](./02_verdaccio.md#usage) in the [edr_evm_napi](../../crates/edr_evm_napi/) directory. You can ignore the warnings about not finding NAPI binaries for other platforms.
```

Comment:
Do you just need to publish the top-level package? Does it transitively publish the platform-specific package?

Reply:
Yes, you just need to publish the top-level package and it'll transitively publish the specific packages.